tests/test_samplers.py

```diff
@@ -0,0 +1,209 @@
+# -*- coding: utf-8 -*-
+
+"""Test that samplers can be executed."""
+
+import unittest
+from typing import Any, MutableMapping
+
+import numpy
+import torch
+import unittest_templates
+
+from pykeen.datasets import Nations
+from pykeen.sampling import BasicNegativeSampler, BernoulliNegativeSampler, NegativeSampler, PseudoTypedNegativeSampler
+from pykeen.training.schlichtkrull_sampler import GraphSampler, _compute_compressed_adjacency_list
+from pykeen.triples import SLCWAInstances, TriplesFactory
+from pykeen.typing import MappedTriples
+
+
+# TODO this whole file needs to be deleted after moving the new tests for the pseudo-typed sampler over
```

Comment:
just keep this file long enough to fix new one in `/tests/samplers/test_negative_samplers.py`